File: src/main/java/com/upokecenter/text/encoders/EncodingUtf16LE.java
package com.upokecenter.text.encoders;

import java.io.*;
import com.upokecenter.util.*;
import com.upokecenter.text.*;

  public class EncodingUtf16LE implements ICharacterEncoding {
    public ICharacterDecoder GetDecoder() {
      return EncodingUtf16.GetDecoder2(0);
    }

    public ICharacterEncoder GetEncoder() {
      return EncodingUtf16.GetEncoder2(0);
    }
  }

File: Encoding/PeterO/Text/Encoders/EncodingUtf16LE.cs
using System;
using System.IO;
using PeterO;
using PeterO.Text;

namespace PeterO.Text.Encoders {
  internal class EncodingUtf16LE : ICharacterEncoding {
    public ICharacterDecoder GetDecoder() {
      return EncodingUtf16.GetDecoder2(0);
    }

    public ICharacterEncoder GetEncoder() {
      return EncodingUtf16.GetEncoder2(0);
    }
  }
}
